docs/dev.md

```diff
@@ -114,33 +140,36 @@ running make (e.g. `export DEV_PROVIDER=azure`).
 6. Wait for infrastructure to be provisioned and the cluster to be deployed. You
    may watch the process with the `./bin/clusterctl describe` command. Example:
 
-```bash
-export KUBECONFIG=~/.kube/config
+   ```bash
+   export KUBECONFIG=~/.kube/config
 
-./bin/clusterctl describe cluster <clusterdeployment-name> -n hmc-system --show-conditions all
-```
+   ./bin/clusterctl describe cluster <clusterdeployment-name> -n hmc-system --show-conditions all
+   ```
 
-> [!NOTE]
-> If you encounter any errors in the output of `clusterctl describe cluster` inspect the logs of the
-> `capa-controller-manager` with:
-> ```bash
-> kubectl logs -n hmc-system deploy/capa-controller-manager
-> ```
-> This may help identify any potential issues with deployment of the AWS infrastructure.
+   > [!NOTE]
```

Comment:
I think the format here is broken. Check again.

Reply:
Yes. I just verified. This is not supported on my vs-code preview while Github seems to support it.
Reverting this change.